test/e2e/data/infrastructure-oci/v1beta1/cluster-template-cluster-identity/cluster-identity.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: OCIClusterIdentity
+metadata:
+  name: cluster-identity
```

Comment:
Should this be called something like `cluster-user-principal-identity`? I wasn't sure if you envisioned doing a `instance-principal` test in the future or not.

Reply:
fixed

Reply:
fixed